CONTRIBUTING.md

```diff
@@ -96,6 +96,31 @@ Some components may require specific setup steps. Please refer to their respecti
 - [Node.js Binding](bindings/nodejs/CONTRIBUTING.md)
 - [Python Binding](bindings/python/CONTRIBUTING.md)
 
+## Build from source
```

Comment:
I prefer to remove this section since it's not suitable for contributor.

Reply:
Make sense.